Update lessons 1-4 (#86)
SAFEHR-data/omop-carpentries: episodes/01_what_is_omop.Rmd

  select(concept_name)
```

> **CODING_NOTE**: We use `filter` to identify the row we want and `select` to choose the column we want. These are functions from the dplyr package that can make code clearer. This is because we are working with a database connection. If we were working with a local R object we could use base R code: `cdm$concept$concept_name[cdm$concept$concept_id == 9201]` to get the same result.

Reviewer: added a bit more detail
```suggestion
**CODING_NOTE**: We use `filter` to identify the row(s) we want and `select` to choose the column(s) we want. These are functions from the dplyr package that can make code clearer by chaining instructions together and can be used for a local R object or database connection. If we were working with a local R object we could use also base R code: `cdm$concept$concept_name[cdm$concept$concept_id == 9201]` to get the same result - however this would command would not work with a database connection.
```

### A useful function

> Finding the humanly readable name for a `concept_id` will be a useful function. We can create a function `get_concept_name()` that takes the 'cdm' object and a `concept_id` as input and returns the `concept_name`.

Reviewer: ```suggestion
Finding the humanly readable name for a `concept_id` will be a useful function. We can create a function `get_concept_name()` that takes the 'cdm' object and a `concept_id` as an input and returns the `concept_name`.
```

:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: challenge

> Create the function `get_concept_name()` that takes the the 'cdm' object and a `concept_id` as input and returns the `concept_name`.

Reviewer: ```suggestion
Create the function `get_concept_name()` that takes the 'cdm' object and a `concept_id` as an input and returns the `concept_name`.
```
